frontend/src/metabase/metadata/components/FieldOrderSidesheet/lib.ts

```diff
@@ -2,6 +2,7 @@ import _ from "underscore";
 
 import { NULL_DISPLAY_VALUE } from "metabase/lib/constants";
 import { type IconName, isValidIconName } from "metabase/ui";
+// eslint-disable-next-line no-restricted-imports
```

Comment:
This file will be removed when [the project](https://linear.app/metabase/project/up-level-admin-metadata-editing-0399213bee40/issues) is done.